src/auth/auth.service.ts

```diff
@@ -64,7 +64,7 @@ export class AuthService {
 
     const user = await this.findUserByEmail(email);
 
-    if (await bcrypt.compare(password, user.password)) {
+    if (user && (await bcrypt.compare(password, user.password))) {
```

Comment:
Khúc này not found user nó k break ngay từ bên kia à e?
Sao vẫn phải check user?

Reply:
Dạ nó vẫn work bth ạ, do e test mà quên xóa ạ.